workflows/rnafusion.nf

```diff
@@ -39,34 +28,6 @@ ch_gtf = Channel.fromPath(params.gtf).map { it -> [[id:it.Name], it] }.collect()
 ch_transcript = Channel.fromPath(params.transcript).map { it -> [[id:it.Name], it] }.collect()
 ch_fai = Channel.fromPath(params.fai).map { it -> [[id:it.Name], it] }.collect()
 
-def checkPathParamList = [
-    params.fasta,
-    params.fai,
-    params.gtf,
-    params.transcript,
-]
```

Comment:
As I have two branches of the pipelines, one that generates the refs and one that detects fusion, but the same validation for both, if I set the refs to "exists": true, an error will be thrown while generating the references, there might be a more elegant way to avoid this, it will be investigated for the next release